### ISSUE TITLE
@extend not working in 1_btns.scss file

### ISSUE THREAD

**Issue by YannickMol** [peiche/gatsby-starter-codyhouse#157]
Hi there, me again! Thanks for updating the starter to CodyFrame v3, much appreciated! I am facing one issue with the [Button Group](https://codyhouse.co/ds/components/app/button-group) component, which is giving the following error upon build. Would you happen to have the solution to this?

```
SassError: The target selector was not found.
Use "@extend .btns__btn--selected !optional" to avoid this error.
    ╷
104 │     @extend .btns__btn--selected;
    │     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
  src\styles\components\_1_btns.scss 104:5  root stylesheet
```

**Comment by peiche** [comment]
Hi, unfortunately I can't reproduce this error. My steps:

1. Create the file `_1_btns.scss` in `src/styles/components`. Paste the SCSS from the [button group component](https://codyhouse.co/ds/components/app/button-group).
2. Add this line in the file `src/styles/components/_index.scss`:
    ```
    @use '1_btns' as *;
    ```
3. Run `gatsby develop`. (I also ran `gatsby build` to confirm that a production build works.)

A couple thoughts:
- I had initially neglected to update the component index to use `@use` instead of `@import`. Are you doing that?
- Gatsby's validation didn't like it when I just had `@use` without the `as` keyword. To eliminate errors, I had to add the `as *` at the end.
- Unfortunately I can't commit my changes that include the button group component. The [Codyhouse license](https://codyhouse.co/license#components) forbids it.
